stubs/AMQP.php

```diff
@@ -66,11 +66,16 @@
 
 /**
  * When publishing a message, the message must be routed to a valid queue. If it is not, an error will be returned.
+ * Note: This flag does not currently work in pecl-amqp, and WILL cause undefined behavior if you use it.
+ * @link https://bugs.php.net/bug.php?id=62786
```

Comment:
This is broker-specific (I guess in your case - RabbitMQ specific). Other AMQP 0.9.1 brokers may still use it.